calblueprint/siren
# Configured fonts + spacing

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Configured PT Sans google font + app wide spacing
Established styling convention 

## Coverage 
- [x] import google fonts 
- [x] fix padding/margin styling of page 
- [x] test font weights 

## How can the reviewer test your code? 👩‍💻
`npm start`, see `ScreenStyles.tsx` for styling convention, usage of font + spacing 

## Any bugs you encountered or still having trouble with? 🐛
- Tried debugging fonts using React-Native-Paper for a long time before realizing their own site's example doesn't function 😤 
- [JSON value of type NSString cannot be converted](https://github.com/vitalets/react-native-extended-stylesheet/issues/97) => cannot use `mv` to set dimensions in React Native. 


## Resources 📔
[PT Sans === Myriad Pro (lol) ](http://joelcrawfordsmith.com/closest-font/font/myriad-pro)
[Google Fonts for Expo ](https://github.com/expo/google-fonts)
[Loading fonts once ](https://stackoverflow.com/questions/44310100/using-custom-font-in-react-native-with-expo-loading-font-every-time)
[Expo example ](https://snack.expo.dev/@pieamo/using-expo-custom-font)

🧜‍♀️ cc: @oahnh  @phoebeli23
